src/main/kotlin/pet/project/app/dto/book/BookMapper.kt

```diff
@@ -6,7 +6,7 @@ import pet.project.app.model.Book
 object BookMapper {
 
     fun Book.toDto() = ResponseBookDto(
-        id.toHexString(),
+        id?.toHexString() ?: "None",
```

Comment:
```suggestion
        id!!.toHexString(),
```
It this case it is reasonable to use !!